ql/pricingengines/vanilla/analytichestonengine.hpp

```diff
@@ -149,29 +163,16 @@ namespace QuantLib {
                                   Real& value,
                                   Size& evaluations);
 
+        Real priceVanillaPayoff(
+           const ext::shared_ptr<PlainVanillaPayoff>& payoff,
+           const Date& maturity) const;
+
+        Real priceVanillaPayoff(
+           const ext::shared_ptr<PlainVanillaPayoff>& payoff, Time maturity) const;
+
```

Comment:
Do these need to be public?

Reply:
I think we need something to put doCalcuation() users off till later. It is also used in a market model test case.